lib/connection/process.coffee

```diff
@@ -90,12 +90,19 @@ module.exports = jlprocess =
 
   spawnJulia: (port, cons) ->
     if process.platform is 'win32' and atom.config.get("julia-client.enablePowershellWrapper")
-      @useWrapper = parseInt(child_process.spawnSync("powershell", ["-NoProfile", "$PSVersionTable.PSVersion.Major"]).output[1].toString()) > 2
+      @useWrapper = parseInt(child_process.spawnSync("powershell",
+                                                    ["-NoProfile", "$PSVersionTable.PSVersion.Major"])
+                                          .output[1].toString()) > 2
       if @useWrapper
+        # get a random port between 1000 and 20_000; this may not be free, of course,
+        # but hopefully not very often
+        @wrapPort = Math.round(Math.random()*(20000-1000)+1000)
```

Comment:
It'd be great to do this in a smarter way, i.e. by actually checking if we have a free port. Couldn't figure out how to do that synchronously though, and any async calls will cause [this](https://github.com/JunoLab/atom-julia-client/blob/f52266ba7603c62c9aabd273556283fdd5169e58/lib/connection/process.coffee#L76-L89) to fail (or have `@useWrapper` undefined).